### ISSUE TITLE
Update

### ISSUE THREAD

**Issue by Danil67556** [pWn3d1337/Techguns#273]
Will there be an update of this mod? Since most kravts are unavailable.

**Comment by JAVAHTML1** [comment]
Yes. That would be great because I have used Techguns on 1.12, but moved to 1.19.2. Will this be updated? Great mod wish it was for 1.19.2. Doesn't seem to get many updates.
